examples/twittermap/web/app/controllers/TwitterMapApplication.scala

```diff
@@ -91,8 +101,70 @@ class TwitterMapApplication @Inject()(val wsClient: WSClient,
     }
   }
 
+
+  object LiveTweetActor {
+    def props(out: ActorRef) = Props(new LiveTweetActor(out))
+  }
+
+  class LiveTweetActor(out: ActorRef) extends Actor {
+    override def receive = {
+      case msg: JsValue =>
```

Comment:
Add concise comments about the logic of the retrieving, e.g. you use center of the location and calculate how big a radius for this keyword query, etc.